### ISSUE TITLE
Generate OPML file

### ISSUE THREAD

**Issue by roaldarbol** [sneakers-the-rat/journal-rss#22]
For multi-feed export we need to generate an OPML file, so we need functionality to generate the file content.

I have previously done that in R (https://github.com/roaldarbol/rss-journals/blob/e4c75c0179ef15d2f630d905a383e35a8d969e14/R/functions.R) for a similar project, so we can just reimplement that in python. I can take a swing at it one of the coming days.

**Comment by sneakers-the-rat** [comment]
yes! I think we will want to do something similar to the `fastapi_rss` package we're using for the feeds, make a pydantic model and a renderer method so we can use fastapi's automatic documentation and whatnot, but from your example it looks like that will be pretty straightforward :)

**Comment by roaldarbol** [comment]
The OPML structure is super simple, so that part is not that tricky! I've never used pydantic before, but maybe a good time to learn that. Otherwise feel free to take a swing at it - I'll have to work on my actual PhD today unfortunately 😅
